CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# dbt_fivetran_log v2.2.0
+
+[PR #154](https://github.com/fivetran/dbt_fivetran_log/pull/154) includes the following updates:
+
+## Breaking Change for dbt Core < 1.9.5
+> *Note: This is not relevant to Fivetran Quickstart users.*
+Migrated `freshness` from a top-level source property to a source `config` in alignment with [recent updates](https://github.com/dbt-labs/dbt-core/issues/11506) from dbt Core. This will resolve the following deprecation warning that users running dbt >= 1.9.5 may have received:
+
+```
+[WARNING]: Deprecated functionality
+Found `freshness` as a top-level property of `fivetran_platform` in file
+`models/src_fivetran_platform.yml`. The `freshness` top-level property should be moved
+into the `config` of `fivetran_platform`.
+```
+
+**IMPORTANT:** Users running dbt Core < 1.9.5 will not be able to utilize freshness tests in this release or any subsequent releases, as older versions of dbt will not recognize freshness as a source `config` and therefore not run the tests.
+
+If you are using dbt Core < 1.9.5 and want to continue running TikTok Ads freshness tests, please elect **one** of the following options:
+  1. (Recommended) Upgrade to dbt Core >= 1.9.5
+  2. Do not upgrade your installed version of the `fivetran_platform` package. Pin your dependency on v2.1.0 in your `packages.yml` file.
+  3. Utilize a dbt [override](https://docs.getdbt.com/reference/resource-properties/overrides) to overwrite the package's `fivetran_platform` source and apply freshness via the [old](https://github.com/fivetran/dbt_fivetran_log/blob/v2.1.0/models/staging/src_fivetran_platform.yml#L11-L13) top-level property route. This will require you to copy and paste the entirety of the `src_fivetran_platform.yml` [file](https://github.com/fivetran/dbt_fivetran_log/blob/v2.1.0/models/staging/src_fivetran_platform.yml#L15-L265) and add an `overrides: fivetran_platform_source` property.
```

Comment:
```suggestion
  3. Utilize a dbt [override](https://docs.getdbt.com/reference/resource-properties/overrides) to overwrite the package's `fivetran_log` source and apply freshness via the [old](https://github.com/fivetran/dbt_fivetran_log/blob/v2.1.0/models/staging/src_fivetran_platform.yml#L11-L13) top-level property route. This will require you to copy and paste the entirety of the `src_fivetran_platform.yml` [file](https://github.com/fivetran/dbt_fivetran_log/blob/v2.1.0/models/staging/src_fivetran_platform.yml#L15-L265) and add an `overrides: fivetran_log` property.
```

Since the override is the package name per the dbt_project.yml. This would need to reference log and not platform